build.mjs

```diff
@@ -132,8 +137,10 @@ Metalsmith(__dirname)
         renderer: renderer
     }))
     .use(contents())
+    .use(tutorials('tutorials')())
     .use(permalinks({
-        pattern: ':filename'
+        duplicates: 'error'
+        // pattern: ':filename'
```

Comment:
Maybe just delete this line now?

Reply:
If you are happy with that, sure.  The `duplicates` thing is a handy addition that saved my bacon during testing.

Reply:
commented code removed in 8020947